sabnzbd/downloader.py

```diff
@@ -55,6 +55,8 @@
 _BPSMETER_UPDATE_DELAY = 0.05
 # How many articles should be prefetched when checking the next articles?
 _ARTICLE_PREFETCH = 20
+# Minimum decoder or assembler queue level before slowing down download. 1 = full
+_QSLOWDOWN_LEVEL = 0.6
```

Comment:
Could you give it a less cryptic name? Like `QUEUE_THRESHOLD` or `MAX_QUEUE_LEVEL`?

Reply:
I struggled to come up with anything because it's not an absolute threshold. I changed it to `SOFT_QUEUE_LEVEL` and put it with the other queue limit variables in `constants.py`.